test/factories.ts

```diff
@@ -86,7 +86,13 @@ export const radioFactory = fieldFactoryForType('radio')
 export const optionSelectFactory = fieldFactoryForType('optionSelect')
   .attrs({ options: attrOptions });
 
+export const objectSearchFactory = fieldFactoryForType('objectSearch')
+  .attrs({
+    endpoint: '/endpoint/',
+  });
+
 export const objectSearchCreateFactory = fieldFactoryForType('objectSearchCreate')
+  .extend(objectSearchFactory)
```

Comment:
Wouldn't this extension override the type of `objectSearchCreateFactory`?

Reply:
I suppose so?

Reply:
Seems fine